### ISSUE TITLE
Feature request: Allow empty scope, but if scope exists must be listed in config

### ISSUE THREAD

**Issue by scouten-adobe** [KeisukeYamashita/commitlint-rs#355]
I'd like to be able to allow an empty scope, _but_ also restrict the scope to a specific list if a scope appears.

I've configured my `.commitlintrc.yml` with the following:

```yaml
  scope:
    level: error
    options:
      - rp_sandbox_a
      - rp_sandbox_b
      - rp_sandbox_c

  # Scope may be empty
  scope-empty:
    level: ignore
```

… but a commit with no scope fails the `scope` rule.

**Comment by KeisukeYamashita** [comment]
Sorry for the late reply, I'll check if it reproduces.

**Comment by KeisukeYamashita** [comment]
Thank you for filing this issue!
Let me check soon!

**Comment by KeisukeYamashita** [comment]
Released [v0.1.12](https://github.com/KeisukeYamashita/commitlint-rs/releases/tag/v0.1.12) 🚀

**Comment by scouten-adobe** [comment]
Sorry for long delay in catching this, but the fix looks good. Thank you!
